src/reynir_correct/errtokenizer.py

```diff
@@ -2402,6 +2425,32 @@ def number_error(
             at_sentence_start = False
 
 
+def late_fix_merges(
+    token_stream: Iterable[CorrectToken],
+    db: GreynirBin,
+    token_ctor: TokenCtor,
+    only_ci: bool,
+) -> Iterator[CorrectToken]:
+    """Annotate tokens where error annotation
+    has disappeared due to token merging"""
+    for token in token_stream:
+        if (
+            token.original
+            and token.txt.strip() != token.original.strip()
+            and isinstance(token, CorrectToken)
+            and not token.error
+        ):
+            token.set_error(
+                SpellingError(
+                    "005",
+                    "Skrifa á '{}' sem '{}'".format(token.original.strip(), token.txt),
```

Comment:
Is "Rita" better than "Skrifa" here? Also, for consistency, do token.txt.strip()

Reply:
Done and done